packages/paste-website/src/pages/content/product-style-guide.mdx

```diff
@@ -192,56 +198,106 @@ Coming in the next release
 - *Behavior, dialed, color, artifact, center, license* (for both verb and noun), *-ize* endings, *program* (for all meanings)
 
 #### Bold and italics
-Coming in the next release
+
+##### Bold 
+**Do** use bold type sparingly, to draw attention to the most important word, number, or short phrase in your copy. 
+- Bold type can help a user scan the page more effectively by making landmarks more prominent.
+
+**Do** consider using bold type to refer to UI elements in instructional content. E.g.
+- *Select the location from **Region/State**.*
+- *Select **Continue** to review your order.*
+
+**Don't** use bold multiple times in body copy or for a whole sentence/paragraph. It dilutes its effect.
+- Only **bold the most important information the user needs to retain** and deploy it only if user testing flags a need. 
+- Consider if putting an important sentence on a new line will do a better job of making it stand out than making it bold. 
```

Comment:
This is great in general! I love how the docs themselves are an example of how to best employ these guidelines 🙌